app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1> list of users </h1>
\ No newline at end of file
+<h1>list of users</h1>
+<div class="user">
+<%= render 'partials/user', user: @user %>
```

Comment:
Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB2NSP7SPLVSZL5GNLPARTWUY2ZRPWQ) wireframe for the `user show` page.

Reply:
@SajjadAhmad14 Thanks for that review, let me make the change